vertica/README.md

```diff
@@ -38,6 +38,21 @@ As the metrics for current license usage use the values from the most recent [au
 
 [Restart the Agent][8] to start sending Vertica metrics to Datadog.
 
+#### Authenticating with Kerberos
+
+The Vertica integration supports using Kerberos for authentication. To enable Kerberos authentication,
+you need to manually install the [kerberos][13] library:
+
+```bash
+sudo -Hu dd-agent /opt/datadog-agent/embedded/bin/pip install kerberos==1.3.1
+```
```

Comment:
```suggestion
   ```bash
   sudo -Hu dd-agent /opt/datadog-agent/embedded/bin/pip install kerberos==1.3.1
   ```
```

Reply:
that comment didn't turn out right, but basically add three spaces in front of this codeblock, so make the ordered list alignment work